aleph/index/xref.py

```diff
@@ -89,7 +90,13 @@ def iter_matches(collection, authz):
         authz_query(authz, field="match_collection_id"),
     ]
     query = {"query": {"bool": {"filter": filters}}, "_source": XREF_SOURCE}
-    for res in scan(es, index=xref_index(), query=query):
+    for res in scan(
+        es,
+        index=xref_index(),
+        query=query,
+        scroll=SETTINGS.XREF_SCROLL,
+        size=SETTINGS.XREF_SCROLL_SIZE,
+    ):
```

Comment:
@stchris I’m not entirely sure, but isn’t the relevant method `iter_entities`?

https://github.com/alephdata/aleph/blob/main/aleph/index/entities.py#L59

Reply:
You're very much right. For consistency we might leave this in, because it also relates to xref operations.